source/query-with-natural-language/query-with-natural-language.txt

```diff
@@ -11,7 +11,54 @@ Query with Natural Language
    :backlinks: none
    :depth: 2
 
-Query with Natural Language
+You can use MongoDB Compass to generate filter queries and aggregations 
+using natural language. Compass uses AI to generate filter queries based 
+on prompts you provide. Querying with natural language can be a helpful 
```

Comment:
(minor tweaks)
The first sentence reads a little clunky to me. What about a slight tweak. Something like:
You can use MongoDB Compass to ask natural language questions about your data. Compass uses AI to generate filter queries and aggregations based on the prompts you provide.